packages/demo/components/pages/apis/transaction/builder/txIn.tsx

```diff
@@ -5,7 +5,7 @@ export default function TxIn() {
   return (
     <Section
       sidebarTo="txIn"
-      header="Set pubkey input for transaction"
+      header="Set the transaction inputs"
```

Comment:
This change is logically more correct, but might not be friendly for users as a tech doc perspective (coz its not suggesting pub key input could be set this way).